wcivf/apps/elections/templates/elections/includes/_single_ballot.html

```diff
@@ -142,13 +142,14 @@ <h5>Citizenship</h5>
     <div class="card">
       <h3>
         <span aria-hidden="true">📅</span>
-        Election events near you
+        Husting events
```

Comment:
In the past we've avoided using "hustings" as it's a word that I think most people understands. I would be nice to use language that explains what happens, and then include the jargon word alongside it.